pythonforandroid/recipes/joblib/__init__.py

```diff
@@ -0,0 +1,14 @@
+from pythonforandroid.recipe import PythonRecipe
+
+
+class ThisRecipe(PythonRecipe):
```

Comment:
Can you follow the naming of the recipe class like the ones we already have on `python-for-android` ?

E.g. `JoblibRecipe`